sklearn/preprocessing/label.py

```diff
@@ -799,6 +799,24 @@ class MultiLabelBinarizer(BaseEstimator, TransformerMixin):
     >>> list(mlb.classes_)
     ['comedy', 'sci-fi', 'thriller']
 
+    The following counter-example should be a reminder to new users to pass in
```

Comment:
Maybe I'd say "A common mistake is to pass in a list, which leads to the following issue:" or something like that?